apps/ui/src/components/sidebar/view.tsx

```diff
@@ -59,18 +29,22 @@ export function SideBarView() {
       )}
       {shouldDisplayHeader ? (
         <div className="flex flex-col gap-2">
-          <h2 className="font-bold">
-            {upperFirst(sidebarContext.view?.type || "")}
+          <h2 className="font-bold first-letter:uppercase">
+            {sidebarContext.view?.type || ""}
           </h2>
-          <p className="text-sm">{description}.</p>
+          <p className="text-sm">
+            {
+              sideBarComponentMapper(
+                Number(schemasContext?.schemaRef.current?.hash.length) > 0,
+              )[viewType].description
+            }
```

Comment:
This is unreadable and needs work. Maybe try defining some vars with meaningful names above and use them here.